config/routes.rb

```diff
@@ -5,4 +5,6 @@
   resources :jobs, only: [:index]
 
   root "jobs#index"
+
+  get "appInspector", to: "app_inspector#index"
```

Comment:
I don't believe URLs are case sensitive. Let's add a separator here. I used `_` but `-` is just fine too if you prefer.

```suggestion
  get "app_inspector", to: "app_inspector#index"
```